connectors/sqs/element-templates/aws-sqs-inbound-intermediate-connector.json

```diff
@@ -0,0 +1,230 @@
+      "id": "messagePollingProperties",
+      "label": "Message polling properties"
+    },
+    {
+      "id": "input",
+      "label": "Use next attribute names for activation condition"
+    },
+    {
+      "id": "activation",
+      "label": "Activation"
+    },
+    {
+      "id": "variable-mapping",
+      "label": "Variable Mapping"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "value": "io.camunda:connector-aws-sqs-polling:1",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.type"
+      }
+    },
+    {
+      "type": "Hidden",
+      "generatedValue": {
+        "type": "uuid"
+      },
+      "binding": {
+        "type": "bpmn:Message#property",
+        "name": "name"
+      }
+    },
+    {
+      "label": "Access key",
+      "description": "Provide AWS IAM access key that has permission to send to desired SQS",
+      "group": "authentication",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "authentication.accessKey"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Secret key",
+      "description": "Provide AWS IAM secret key that has permission to send to desired SQS",
+      "group": "authentication",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "authentication.secretKey"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Queue URL",
+      "description": "Specify the URL of the SQS queue where you would like to send message to",
+      "group": "queueProperties",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "queue.url"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "pattern": {
+          "value": "(^https?://.+)|(^arn:.+)(^secrets\\..+)",
+          "message": "Must be an queue URL or ARN or a secret.XYZ"
+        }
+      }
+    },
+    {
+      "label": "Region",
+      "description": "Specify the AWS region of your queue",
+      "group": "queueProperties",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "queue.region"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Polling wait time",
+      "description": "The duration (in seconds) for which the call waits for a message to arrive in the queue before returning. See <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/aws-sqs/\" target=\"_blank\">documentation</a> for details",
+      "group": "messagePollingProperties",
+      "type": "String",
+      "value": "1",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "queue.pollingWaitTime"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "pattern": {
+          "value": "(^[0-9]?$|^1[0-9]$|^20$)|(^secrets\\..+)",
```

Comment:
Please take a look if this applies https://github.com/camunda/connectors-bundle/pull/624